internal/pkg/term/selector/selector.go

```diff
@@ -748,24 +757,24 @@ func (s *LocalWorkloadSelector) Service(msg, help string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("retrieve services from workspace: %w", err)
 	}
-	storeServiceNames, err := s.ConfigSelector.workloadLister.ListServices(summary.Application)
+	storeServices, err := s.ConfigSelector.workloadLister.ListServices(summary.Application)
 	if err != nil {
 		return "", fmt.Errorf("retrieve services from store: %w", err)
 	}
-	serviceNames := filterWlsByName(storeServiceNames, wsServiceNames)
-	if len(serviceNames) == 0 {
-		return "", errors.New("no services found")
-	}
-	if len(serviceNames) == 1 {
-		log.Infof("Only found one service, defaulting to: %s\n", color.HighlightUserInput(serviceNames[0]))
-		return serviceNames[0], nil
+	options, err := s.generateOptions(storeServices, wsServiceNames, svcWorkloadType)
+	if err != nil {
+		return "", err
 	}
 
-	selectedServiceName, err := s.prompt.SelectOne(msg, help, serviceNames, prompt.WithFinalMessage(svcNameFinalMsg))
+	if len(options) == 1 {
+		log.Infof("Only found one service, defaulting to: %s\n", color.HighlightUserInput(options[0].Value))
```

Comment:
Nit: this is totally not on you, but I wonder if we can take this opportunity to change all of these from "Only found" to "Found only"...? 😬